binder/src/main/java/io/grpc/binder/SecurityPolicy.java

```diff
@@ -43,7 +43,8 @@ protected SecurityPolicy() {}
   /**
    * Decides whether the given Android UID is authorized. (Validity is implementation dependent).
    *
-   * <p><b>IMPORTANT</b>: This method may block for extended periods of time.
+   * <p><b>IMPORTANT</b>: This method may block for extended periods of time. For slow or
```

Comment:
Let's leave this comment out for now, since AsyncSecurityPolicy will be experimental for a time.

We can enforce this preference internally since we'll able to handle any changes to the new API, but 3p users won't have the luxury.